scripts/find_equivalent_ext_require_blocks.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+#
+# Copyright 2024 <your actual name, or other organization holding copyright>
```

Comment:
Please replace "your actual name, or other organization holding copyright" with, well, what it says :-)

Reply:
I believe my pseudonym should be fine, I contributed to GCC under this name, the Free Software Foundation has the paperwork for that. I don't know if that's compatible or whatnot, but hopefully it's fine. If not I guess I can use my legal name but I would like to avoid that if possible.